qa/src/test/java/org/apache/brooklyn/qa/load/AbstractLoadTest.java

```diff
@@ -161,8 +161,8 @@ protected ManagementContext setUpPlatform() {
         // Create management node
         persistenceDir = Files.createTempDir();
         launcher = BrooklynLauncher.newInstance()
-                .persistMode(PersistMode.CLEAN)
-                .highAvailabilityMode(HighAvailabilityMode.MASTER)
+                .persistMode(doPersistence() ? PersistMode.CLEAN : PersistMode.DISABLED)
+                .highAvailabilityMode(doPersistence() ? HighAvailabilityMode.MASTER : HighAvailabilityMode.DISABLED)
```

Comment:
I think for a flag, a field would be cleaner than a method to override

Reply:
due to how tests are structured and annotated methods run by the framework, methods for this sort of thing are easier to work with i find.  happy if you want to spike an alternative with fields but i doubt it's better.

Reply:
Good point, didn't think of the complexity due to the framework and annotation. Your solution works so let's go with that, we can iterate on it later on